src/ensembl/src/shared/components/communication-framework/contact-us/contact-us-form/initial/ContactUsInitialForm.tsx

```diff
@@ -0,0 +1,247 @@
+    dispatch({ type: 'remove-file', payload: index });
+  };
+
+  const handleSubmit = useCallback((e: React.SyntheticEvent) => {
+    e.preventDefault();
+    stateRef.current && submitForm(stateRef.current);
+    setIsSubmitted(true);
+  }, []);
+
+  const isFormValid = validate(state);
+
+  if (isSubmitted) {
+    return <SubmissionSuccess />;
+  }
+
+  return (
+    <div className={commonStyles.container}>
+      <div className={commonStyles.grid}>
+        <p className={commonStyles.advisory}>
+          <span>All fields are required unless marked optional</span>
+          <span>Second line</span>
+          <span>Third line</span>
+        </p>
+      </div>
+      <form
+        className={commonStyles.grid}
+        autoComplete="off"
+        onSubmit={handleSubmit}
+      >
+        <label htmlFor="name" className={commonStyles.label}>
+          Your name
+        </label>
+        <ShadedInput id="name" value={state.name} onChange={onNameChange} />
+
+        <label htmlFor="email" className={commonStyles.label}>
+          Your email
+        </label>
+        <ShadedInput id="email" value={state.email} onChange={onEmailChange} />
+
+        <label htmlFor="subject" className={commonStyles.label}>
+          Subject
+        </label>
+        <ShadedInput
+          id="subject"
+          value={state.subject}
+          onChange={onSubjectChange}
+        />
+
+        <label htmlFor="message" className={commonStyles.label}>
+          Message
+        </label>
+        <ShadedTextarea
+          id="message"
+          value={state.message}
+          onChange={onMessageChange}
+          className={commonStyles.textarea}
+        />
+
+        <div className={commonStyles.upload}>
+          {state.files.map((file, index) => (
+            <UploadedFile
+              key={index}
+              file={file}
+              onDelete={() => deleteFile(index)}
+              classNames={{ wrapper: commonStyles.uploadedFile }}
+            />
+          ))}
+          <Upload
+            label="Click or drag a file here to upload"
+            callbackWithFiles={true}
+            allowMultiple={false}
+            onChange={onFileChange}
+          />
+        </div>
+
+        <div className={commonStyles.submit}>
+          {exceedsAttachmentsSizeLimit(state) && (
+            <span className={commonStyles.errorText}>
+              Attachment(s) exceed 10 MB
+            </span>
+          )}
+
+          <PrimaryButton type="submit" isDisabled={!isFormValid} onClick={noop}>
+            Submit
+          </PrimaryButton>
+        </div>
+      </form>
+    </div>
+  );
+};
+
+const validate = (formState: State) => {
+  return (
+    areMandatoryFieldsFilled(formState) &&
+    !exceedsAttachmentsSizeLimit(formState)
+  );
+};
+
+const areMandatoryFieldsFilled = (formState: State) => {
+  return (['name', 'email', 'subject', 'message'] as const).every(
+    (field) => formState[field]
+  );
+};
```

Comment:
Don't we also need to validate if the `email` is valid or not?

Reply:
Not sure yet; also not sure how we will be showing this in the UI. Let's ignore this for now.